lib/cosmos/tools/tlm_viewer/tlm_viewer.rb

```diff
@@ -433,9 +433,8 @@ def closeEvent(event)
           event.ignore()
         else
           # Close any open screens
-          script_disconnect()
```

Comment:
No.  Because DataViewer uses this to Stop, but it might Start again.   shutdown_cmd_tlm is fatal, but script_disconnect is not.